packages/core/src/application/application-factory.ts

```diff
@@ -70,16 +70,16 @@ class AppFactory {
         case AppExpress:
           app = container.resolve(appTypeOrMiddlewares as new () => AppExpress);
           app.create(container);
-          return app as IApplicationExpress;
```

Comment:
I can't approve this request to remove the `return app as IApplication` within each switch block, The reason why is because we will have multiple interfaces, as commented in the code you can see that we can possibly have a fastify interface, and we will have much more in the future.

The return has to be constrained within each switch case.